assets/src/scss/layout/navbar/_search.scss

```diff
@@ -157,31 +157,22 @@
     margin-inline-end: 16px;
   }
 
-  @include x-large-and-up {
+  @include bs-xx-large-and-up {
     background-color: var(--top-navigation--color, $white);
     display: inline-block;
     margin-inline-end: 0;
   }
 }
 
 .nav-search-toggle-container {
-  display: none;
+  display: inline-block;
   padding: $sp-1 $sp-1x;
 
-  &.medium-and-less {
```

Comment:
Don't needed since we've removed [this](https://github.com/greenpeace/planet4-master-theme/pull/1738/files#diff-be7863262a0782b0bd09bbc1307c8819fe0260c6e0de6a47f5ae9b3cd958d4f8L46)